Transaction.cs

```diff
@@ -34,4 +39,20 @@ public decimal GetChangeForAccount(string name)
 
         return 0;
     }
+
+    public void UpdateAccounts(Dictionary<string, Account> accounts)
```

Comment:
I'm not sure this function really belongs here, as this is not really something you're doing to the transaction object, it's something you're doing to the accounts dictionary 🙂